### PR TITLE
Fix e2e test error on page transition

### PR DESCRIPTION
**Pull request by jrichter1**

### CODE REVIEW

File: test/system/test-base.js [hunk 1]
```diff
@@ -222,6 +222,15 @@ function systemTest() {
                         fail(product + 'failed to install');
                         done();
                       }
+                    }).catch(function(error) {
+                      if (error.message.indexOf('element is not attached') > -1) {
+                        browser.getLocationAbsUrl().then(function(url) {
+                          if (url !== '/start') {
+                            throw error;
+                            done();
```
Comment (dgolovin) [inline test/system/test-base.js]: This done() call seems never going to happen, because of throw before it.

Comment (jrichter1) [inline test/system/test-base.js]: right

File: test/system/test-base.js [hunk 2]
```diff
@@ -222,6 +222,14 @@ function systemTest() {
                         fail(product + 'failed to install');
                         done();
                       }
+                    }).catch(function(error) {
+                      if (error.message.indexOf('element is not attached') > -1) {
+                        browser.getLocationAbsUrl().then(function(url) {
+                          if (url !== '/start') {
+                            throw error;
+                          }
```
Comment (dgolovin) [inline test/system/test-base.js]: Should  it call done() in case url == '/start'?

Comment (jrichter1) [inline test/system/test-base.js]: Not really, it will just skip into the next iteration and the /start handling is right at the start of that.

Comment (dgolovin) [inline test/system/test-base.js]: LGTM 👍

**Pull request merged**